src/components/security_manager/include/security_manager/crypto_manager_impl.h

```diff
@@ -80,6 +80,15 @@ class CryptoManagerImpl : public CryptoManager {
 
    private:
     void PrintCertInfo();
+
+    /**
+     * @brief Removes disallowed for printing certificate information from input
+     * data
+     * @param in_data input data with certificate information
+     * @return filtered string with allowed for printing information
+     */
+    std::string RemoveDisallowedInfo(X509_NAME* in_data) const;
```

Comment:
@LuxoftAKutsan yes, done in https://github.com/smartdevicelink/sdl_core/pull/1636/commits/299664d8d96664a1e3102685faa8fa9ef1de86c2